vendor/bat-native-ledger/src/bat/ledger/internal/bat_get_media.cc

```diff
@@ -874,6 +874,10 @@ void BatGetMedia::processTwitchMediaPanel(
     const ledger::VisitData& visit_data,
     const std::string& providerType,
     const std::string& publisher_blob) {
+  if (publisher_blob == "ignore") {
```

Comment:
Should we place this in `static_values.h` so we have one place for it?

Reply:
done, moved it into ledger, so we can use it in rewards code if needed